controllers/sr_controller/sr_controller.py

```diff
@@ -137,27 +140,65 @@ def reconfigure_environment(robot_file: Path) -> None:
     os.chdir(str(robot_file.parent))
 
 
+def log_filename(zone_id: int) -> str:
+    now = datetime.datetime.now()
```

Comment:
Can we have a comment here notating that this is explicitly and deliberately both naïve and in the local timezone?

Reply:
log_filename

Reply:
3f376f1